docs/codegen/overview.md

```diff
@@ -8,3 +8,36 @@ each integration scenario in the best way possible. Currently, we support:
 
 * [HCL generation](./generate-hcl.md)
 * [File generation](./generate-file.md)
+
+# Assertions
+
+Assertions can be used in order to fail code generation for one or more stacks
```

Comment:
it only fails code generation? what about orchestration?

Reply:
AFAIK it is only code generation (per spec).